src/SFML/Window/Win32/WglContext.cpp

```diff
@@ -426,15 +433,14 @@ void WglContext::createContext(WglContext* shared, unsigned int bitsPerPixel, co
                           << "disabling comptibility and debug" << std::endl;
 
                 m_settings.attributeFlags = ContextSettings::Default;
-
-                int attributes[] =
-                {
-                    WGL_CONTEXT_MAJOR_VERSION_ARB, static_cast<int>(m_settings.majorVersion),
-                    WGL_CONTEXT_MINOR_VERSION_ARB, static_cast<int>(m_settings.minorVersion),
-                    0,                             0
-                };
-                m_context = wglCreateContextAttribsARB(m_deviceContext, sharedContext, attributes);
             }
+
+            // Append the terminating 0
+            attributes.push_back(0);
+            attributes.push_back(0);
```

Comment:
The attributes are read in pairs... so theoretically the operating system doesn't have to read the second one, but who knows if it really does... There are a lot of broken things in X.